src/ziggurat/metrics.clj

```diff
@@ -2,43 +2,60 @@
   (:require [clojure.tools.logging :as log])
   (:import (com.gojek.metrics.datadog DatadogReporter)
            (com.gojek.metrics.datadog.transport UdpTransport$Builder UdpTransport)
-           (java.util.concurrent TimeUnit)
-           (io.dropwizard.metrics5 MetricRegistry Meter MetricName Histogram)))
+           (io.dropwizard.metrics5 MetricRegistry Meter MetricName Histogram)
+           (java.util.concurrent TimeUnit)))
 
 (defonce ^:private group (atom nil))
 
 (defonce metrics-registry
   (MetricRegistry.))
 
+(defn get-tags
+  [topic-entity-name]
+  (if (nil? topic-entity-name)
+    ["actor" @group]
+    ["actor" @group "topic_name" topic-entity-name]))
```

Comment:
Why is topic_name being hardcoded. Can it be passed from the fn where we report metrics? something like a map which {:topic-name "topic-entity-name"}

Reply:
So that it's flexible enough for user to send any tags. Not just topic entity

Reply:
if this will be the case, can we pass a map instead of a vector to .tagged method of MetricName?
The tagged method is overloaded, one takes in a variadic (even) numbered strings and a Map<String, String>...

Reply:
done